### DISCUSSION TITLE
Wait Node and the until parameter.

### DISCUSSION THREAD

**Discussion by azriel46d** [Netflix/conductor#3056] [Q&A] [answered]
Can the `until` inputParameter be a variable?

I am using a local instance of conductor and attempting the following

```
{
        "type": "WAIT",
        "name":  "Schedule Payment",
        "taskReferenceName": "schedulePayment",
        "inputParameters": {
            "until": "${schedulePayment.output.executionDate}"
        }
}
```
However the UI wont let me save it due to the fact that it is not a valid date. 

I think its a current limitation to the wait node based on this part of the [PR](https://github.com/v1r3n/conductor/blob/739c6152a4ec9b90907fb44cbe894bde7e2c7bbb/core/src/main/java/com/netflix/conductor/core/execution/tasks/Wait.java#L67)

**Comment by manan164** [comment]
Hi @azriel46d , Thanks for pointing this out. [This](https://github.com/Netflix/conductor/blob/a99e939d36d116d58636a4f1c0537c2b60eb7805/core/src/main/java/com/netflix/conductor/validations/WorkflowTaskTypeConstraint.java#L276) change is preventing us to save the workflow definition. I will raise PR to fix this in
some time.

**Comment by manan164** [comment, marked as answer]
Hi @azriel46d This is [PR](https://github.com/Netflix/conductor/pull/3058) for the fix

**Reply by azriel46d** [reply]
That looks great! Thanks for the quick action! 😄

**Comment by azriel46d** [comment]
@manan164 by any chance, is there a timeline for this PR to be merged?

**Reply by manan164** [reply]
Hi @azriel46d , I have addressed all review comments. Waiting for it to get reviewed again and merge.

**Reply by azriel46d** [reply]
Thank you!
